ElementX/Sources/Application/AppSettings.swift

```diff
@@ -144,7 +144,7 @@ final class AppSettings {
     /// Any pre-defined static client registrations for OIDC issuers.
     let oidcStaticRegistrations: [URL: String] = ["https://id.thirdroom.io/realms/thirdroom": "elementx"]
     /// The redirect URL used for OIDC.
-    let oidcRedirectURL: URL = "io.element:/callback"
+    let oidcRedirectURL: URL = InfoPlistReader.main.appScheme
```

Comment:
Errrr, I think you might want this 🙃

```suggestion
    let oidcRedirectURL: URL = "\(InfoPlistReader.main.appScheme):/callback"
```